README.md

```diff
@@ -123,15 +128,16 @@ If a Lord drops their hammer during the Flip phase they are instantly demoted to
 
 ### Uprising
 
-At the start of each game, there is an Uprising counter that begins as 0. A die or some other object may be used to represent this or it can be kept track of mentally by all players as a group.
-
-If the active player is a Lad, they may call "Uprising" before flipping their hammer. If they do, then the turn officially counts as an Uprising. The player must perform a double hammer flip followed by one strike without hesitation (note that they do not get two strikes like they normally would). If they hit their nail with that strike, then the Uprising was successful, and the Uprising counter is incremented by 1. If they drop the hammer, regrip the hammer, or miss their nail the Uprising counter is reset to 0, in which case they must drink and end their turn immediately, passing the hammer to the next player in turn order.
+If there is at least one Lord and no active Uprising nail currently exists, Lads may call "Uprising" at the start of their turn before flipping the hammer. If they do, they place a single Uprising nail somewhere in the stump and hammer it to the same starting depth as all other nails. Uprising rules now apply to the duration of this turn.
```

Comment:
`Uprising rules now apply to the duration of this turn.` should be `Uprising rules now apply for the duration of this turn.`